api/Dockerfile

```diff
@@ -28,6 +28,9 @@ RUN python -m grpc_tools.protoc  \
 
```

Comment:
I randomly checked some of the metrics. The numbers look good. I think it contains a lot of important metrics such as: request-size, -sum, -total, -duration and the mean can be calculated. And all of those are available per endpoint. There are two metrics that I think we could consider to add. What do you think about a metric for:

- Uptime Gauge
- HTTP status code counter (For an overview of errors, timeouts and successful requests)

Reply:
Uptime need to be handled by the host machine/VM.
HTTP status codes are already provided, but do not appear until a request with the status code is given. Also 2xx, 3xx and 5xx are not split but default.